feat: Bring Move into public API (#1107)
TimefoldAI/timefold-solver: docs/src/modules/ROOT/pages/upgrading-timefold-solver/backwards-compatibility.adoc

but we're just not entirely comfortable yet to write their signatures in stone.
====

[#previewFeatures]
= Preview features

> Timefold Solver includes several components which are only available as preview features.

Reviewer: ```suggestion
Timefold Solver may include components that are only available as preview features. Currently available are:
```
